src/html/popup.html

```diff
@@ -45,6 +45,14 @@
 
         <main>
             <div class="container">
+                <div id = "rateSection">
+                    <div class="section">
```

Comment:
Once the user clicks on "Rate it now" button, "<div class="section">" as well as "<div class="divider"></div>" should be hidden. The "rateSection" div wraps the above two divs